.travis.yml

```diff
@@ -11,7 +11,7 @@ addons:
       - libssl-dev
       - libffi-dev
       - python-dev
-install: pip install --upgrade pip tox six
+install: pip install --upgrade pip tox six~=1.12.0
```

Comment:
I think if you lock six version also in azure-cli-core/setup.py besides azure-cli/setup.py. You may not need to put six here anymore.

Reply:
would it be better to keep a note and sync with astroid?

Reply:
resolved offline

Reply:
ok, you can keep it here since six is also a dependency by other packages.